downloads/swan-lake-release-notes/swan-lake-alpha4/swan-lake-alpha4-release-note.md

```diff
@@ -276,32 +287,45 @@ handle secondWord = arrays:get(input, 1);
     }
 
 ```
+
 ##### Kafka Package Updates
- - SecureSocket record is updated.
- - kafka:Producer, kafka:Consumer and kafka:Listener init updated.
+
+ - The `SecureSocket` record is updated.
+
+ - The `kafka:Producer`, `kafka:Consumer`, and `kafka:Listener` inits are updated.
+
 ```ballerina
 kafka:Producer kafkaProducer = check new(kafka:DEFAULT_URL, config);
 kafka:Producer kafkaProducer = check new (bootstrapServers=”localhost:9092”);
 kafka:Producer kafkaProducer = check new(”localhost:9092”);
 // Same for listener and consumer initialization
 ```
+
 ##### NATS Package Updates
```

Comment:
Updates to the nats package (same as above suggestions).

Reply:
Resolving as per https://github.com/ballerina-platform/ballerina-dev-website/pull/2093#discussion_r606886986.